Microsoft.Alm.Authentication/GitHubAuthority.cs

```diff
@@ -44,11 +44,7 @@ public GitHubAuthority(string authorityUrl = null)
 
             Token token = null;
 
-            using (HttpClientHandler handler = new HttpClientHandler()
-            {
-                MaxAutomaticRedirections = 2,
-                UseDefaultCredentials = true
-            })
+            using (HttpClientHandler handler = targetUri.HttpClientHandler)
```

Comment:
@whoisj 
About that.... 

Yes, it should, and to all the [WebRequests] (https://github.com/Microsoft/Git-Credential-Manager-for-Windows/search?utf8=%E2%9C%93&q=webrequest) either.

```
var request = WebRequest.CreateHttp(targetUri);
request.Proxy = targetUri.WebProxy;
```

We did that at first and all looked fine, but we had a problem ahead with the [AuthenticationContext] (https://github.com/Microsoft/Git-Credential-Manager-for-Windows/blob/fd1f3d00d6ece4f6c8eb665609995e218e0ba3b4/Microsoft.Alm.Authentication/AzureAuthority.cs#L77) which is used to acquire the token from azure. :expressionless:  (Tested **only** with a visualstudio.com account)

Finally after failing to make AuthenticationContext's methods to use the damm proxy :pensive:, we rolledback on that change. Without being able to connect first with the authority (no proxy), the program seems to default to basic authentication (correct me if I'm wrong) :confused: , and then it works.

We can send you some tests on that if you want.